src/models/query.rs

```diff
@@ -0,0 +1,115 @@
+use context::GraphQLContext;
+use juniper::{FieldResult, ID};
+use models::*;
+
+pub struct Query;
+
+impl Query {
+    pub fn album(context: &GraphQLContext, id: &str) -> FieldResult<Album> {
+        Album::from_id(context, id)
+    }
+
+    pub fn albums(
+        context: &GraphQLContext,
+        first: i32,
+        after: Option<String>,
+        sort: Option<SortParams>,
+    ) -> FieldResult<Connection<Album>> {
+        NotImplementedErr()
+    }
+
+    pub fn artist(context: &GraphQLContext, id: &str) -> FieldResult<Artist> {
+        Artist::from_id(context, id)
+    }
+
+    pub fn artists(
+        context: &GraphQLContext,
+        first: i32,
+        after: Option<String>,
+        sort: Option<SortParams>,
+    ) -> FieldResult<Connection<Artist>> {
+        NotImplementedErr()
+    }
+
+    pub fn song(context: &GraphQLContext, id: &str) -> FieldResult<Song> {
+        Song::from_id(context, id)
+    }
+
+    pub fn songs(
+        context: &GraphQLContext,
+        first: i32,
+        after: Option<String>,
+        sort: Option<SortParams>,
+    ) -> FieldResult<Connection<Song>> {
+        NotImplementedErr()
+    }
+
+    pub fn playlist(context: &GraphQLContext, id: &str) -> FieldResult<Playlist> {
+        Playlist::from_id(context, id)
+    }
+
+    pub fn playlists(
+        context: &GraphQLContext,
+        first: i32,
+        after: Option<String>,
+        sort: Option<SortParams>,
+    ) -> FieldResult<Connection<Playlist>> {
+        NotImplementedErr()
+    }
+}
+
+graphql_object!(
+    Query: GraphQLContext | &self | {
```

Comment:
Line is broken

Reply:
I'mt not sure why this isn't auto collapsing.